recipes/ner_pre_annotations_openai.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>This allows the model to predict the integer labels during training and inference, instead of predicting the string tags directly.</p><p>-&gt; The model output is composed by integers that index the class, and the dictionary helps to recover the class string.</p><p> </p><p>to make bullet points lighter, you can remove "&nbsp;named entity."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>